Source/Mosa.Compiler.MosaTypeSystem.CLR/Metadata/ClrMetadata.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+namespace Mosa.Compiler.MosaTypeSystem.CLR.Metadata;
+
+internal class ClrMetadata : IMetadata
+{
+	private readonly ClrModuleLoader moduleLoader;
+
+	public ClrMetadata(ClrModuleLoader loader)
```

Comment:
Fix warning

Reply:
The warning for ``moduleLoader`` is due to the name not respecting the convention, but @tgiphil does not want underscores at the beginning of variables for private fields.

Reply:
The warnings are about properties possibly being null, which means they either need to have a value or their type should be nullable.